src/components/Client.tsx

```diff
@@ -0,0 +1,32 @@
+import { useFetchUsersChatsQuery } from "@/services/chats";
+import {
+  addChatObjectToDB,
+  addClientToChatInDB,
+  getChatObject,
+} from "@/services/firebaseChatMethods";
+import { StateType } from "@/types/StateType";
+import { UserObjectType } from "@/types/UserType";
+import { useSelector } from "react-redux";
+
+const Client = ({ clientData }: { clientData: UserObjectType }) => {
+  const user = useSelector((state: StateType) => state.userReducer.user);
```

Comment:
`const selectUser = (state: StateType) => state.userReducer.user`

you can prepare selectors ` useSelector(selectUser)`